### PR TITLE
Redesign pricing page with 3 tiers

### LINKED ISSUE #713: Redesign "Pricing" (3 tears)
### Description

Our top-level pricing display needs some thinking.

Currently - we have 22% of people clicking the "Install" option on the Pricing page:

Screenshot 2023-05-03 at 12 39 08

and only 7% click the "Join FF Cloud" option:

Screenshot 2023-05-03 at 12 39 31

What could we change in order to encourage more users to FF Cloud? We should be able to A/B test these results too to check for validation of improvement.

Interestingly we also get 4% of users clicking the "per Node-RED instance/month" tagline under the actual price, but this does nothing, what extra detail could we provide here?

### Epic/Story

#773

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Design update for the pricing page

## Related Issue(s)

Closes #713 

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/pricing.njk [hunk 1]
```diff
@@ -4,42 +4,157 @@ align: center
...
+                <note class="font-bold text-teal-100" style="display: inline-block;">All of Premium features, plus:</note>
+                <ul class="ff-checklist font-light text-white leading-7 mt-2">
+                    <li>Auditability</li>
+                    <li>Enterprise integration</li>
+                    <li>Quicker support response</li>
+                </ul>
+            </div>
+        </div>
+            
+        {# <div class="pricing-tile pt-4 pb-6 px-8 justify-between middle md:h-full">
```
Comment (ZJvandeWeg) [inline src/pricing.njk:109]: Can we remove commented out code?

Comment (Yndira-FlowForge) [reply src/pricing.njk:109]: For sure, but I'm still working on it!

File: src/pricing.njk [hunk 2]
```diff
@@ -4,42 +4,157 @@ align: center
...
         </div>
-        <div class="flex flex-col sm:flex-row items-center gap-4 mt-4">
-            <div class="sm:w-6/12">
-                <h5>FlowForge Cloud</h5>
-                <p class="mt-2">Get started in less than a minute, fully managed, fully secure.</p>
-                <a class="ff-btn ff-btn--primary uppercase" href="https://app.flowforge.com/account/create" onclick="capture('cta-join', {'position': 'primary'})">Free Trial</a>
+
+            <div class="pricing-tile pb-6 md:h-full middle">
+                <div class="h-8 flex items-center justify-center text-center text-xs font-semibold text-white bg-gray-600 w-auto -mx-7 rounded-t-lg">  
+                    Production ready
```
Comment (ZJvandeWeg) [inline src/pricing.njk:45]: Looking at this, it feels a bit like the other tiers aren't production ready, what do you think?

Comment (ZJvandeWeg) [reply src/pricing.njk:45]: "Get started instantly" works!

File: src/pricing.njk [hunk 3]
```diff
@@ -4,46 +4,114 @@ align: center
...
-                <h2>$0</h2>
-                <label class="sub-label">Apache 2.0 Licensed<label>
-            </div>
-            <p class="mt-4">We are an open-core company, use FlowForge for free, always, with deployment options including Docker and Kubernetes.</p>
-            <a class="ff-btn ff-btn--secondary uppercase" href="/docs/install" onclick="capture('cta-install', {'position': 'primary'})">Install now</a>
-        </div>
+<div class="px-6 pricing-tiles flex flex-col mb-16 md:max-w-screen-lg">
+    <div class="flex px-6 -mb-6 -mt-32 mx-auto text-gray-400 md:max-w-screen-lg md:-mt-40 md:mb-1 md:block">
+        <p class="lead-p max-w-screen-lg m-auto text-center md:max-w-xl md:px-4">
+            Plans for every project. Whether you're an individual contributor, an IIoT manager, or an IT department, you'll find a suitable option to take your Node-RED applications to the next level.
```
Comment (ZJvandeWeg) [inline src/pricing.njk:13]: I think individual contributor is a phrase only managers use, not engineers themself?

```suggestion
            Plans for every project. Whether you're an engineer, an IIoT manager, or an IT department, you'll find a suitable option to take your Node-RED applications to the next level.
```

**Comment by ZJvandeWeg** [comment]
The faded letters, combined with the arrow down makes me click the arrow down. I expected the box to extend down?

Screenshot 2023-05-23 at 14 18 32

File: src/pricing.njk [hunk 4]
```diff
@@ -4,46 +4,114 @@ align: center
...
+            <div class="flex flex-col justify-between items-center text-center mb-4 sm:px-4 md:flex-row md:-mb-0 md:px-0 md:block md:text-left md:i md:items-left">
+                <h4 class="mt-2 pt-4" style="display:inline-block;">Starter </h4> <h5 style="display:inline-block;">(Open Source)</h5>
+                <label class="label">Self-managed</label>
+                <div class="mt-6 items-center md:text-left md:items-left">
+                    <h2>Free</h2>
+                </div>
+                <p class="mt-6 pb-2"><span class="inline font-normal">Open-Source</span> version of the platform, with deployment options including Docker and Kubernetes.</p>
+                </div>
+                <div class="flex flex-col items-center">
+                <a class="ff-btn ff-btn--secondary-outlined uppercase align-baseline w-full" href="/docs/install" onclick="capture('cta-install', {'position': 'primary'})">Install now</a>
```
Comment (ZJvandeWeg) [inline src/pricing.njk:27]: This link is broken, maybe due to Netlify?

File: src/pricing.njk [hunk 5]
```diff
@@ -4,46 +4,114 @@ align: center
...
+                    Production ready
+                </div>
+                <div class="flex flex-col justify-between items-center text-center mb-4 sm:px-4 md:flex-row md:-mb-0 md:px-0 md:block md:text-left md:i md:items-left">
+                    <div class="flex flex-col items-center md:flex-row md:items-end md:justify-between">
+                        <div class="md:w-3/5">
+                            <h4 class="pt-4" style="display: inline-block;">Premium </h4>
+                            <label class="label">FlowForge Cloud</label>
+                        </div>
+                        <div class="annual-text h-full p-1 mt-1 md:w-2/5 md:ml-2">
+                            <a class="text" href='/book-demo'>Contact us for annual billing discounts</a>
```
Comment (ZJvandeWeg) [inline src/pricing.njk:54]:
```suggestion
                            <a class="text" href='/contact-us/'>Contact us for annual billing discounts</a>
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]
Minor questions, but mostly impressed! Much better than before!

File: src/pricing.njk (hunk 3 above)
Comment (Yndira-FlowForge) [reply src/pricing.njk:13]: I was thinking more broadly, not exclusively engineers but hobbyist too...
Taking that into consideration, what do you think? Should we add "engineer" or replace the "individual contributor".

**Comment by Yndira-FlowForge** [comment]
> The faded letters, combined with the arrow down makes me click the arrow down. I expected the box to extend down?
> 
> Screenshot 2023-05-23 at 14 18 32

Hmm... I had my doubts about this. The idea is to convey that those are not all the features, so the user scrolls down, and the chevron was supposed to indicate that the rest of the information is below. But if it made you click it, maybe it would be better to just remove it?

File: src/pricing.njk (hunk 4 above)
Comment (Yndira-FlowForge) [reply src/pricing.njk:27]: I'm not sure, is the same link we have right now on main... And it works locally, for me at least...

**Comment by ZJvandeWeg** [comment]
> But if it made you click it, maybe it would be better to just remove it?

Or make the screen drop to the feature list? With an anchor?

File: src/css/style.css [hunk 6]
```diff
@@ -770,14 +782,14 @@ h4:hover .header-anchor {
     
     .ff-feature-table-section {
         position: relative;
-        margin-bottom: 32px;
+        /* margin-bottom: 32px; */
```
Comment (ZJvandeWeg) [inline src/css/style.css:785]: ✂️

File: src/css/style.page.css [hunk 7]
```diff
@@ -151,53 +162,137 @@
...
+
     .pricing-tile label {
-        font-weight: 200;
-        color: theme(colors.gray.300);
-        font-size: 0.875rem;
+        font-weight: 300;
+        display: block;
+        color: theme(colors.white);
+        font-size: 16px;
+        /* margin-top: -3px; */
```
Comment (ZJvandeWeg) [inline src/css/style.page.css:218]: ✂️

File: src/_includes/feature_lists/cloud.njk [hunk 8]
```diff
@@ -4,117 +4,117 @@
         rows: [{
             id: 'hosting',
             label: "Hosting",
-            values: ['Self-Managed', 'FlowForge or Self-Managed'],
+            values: ['Self-Managed', 'FlowForge or Self-Managed', 'Self-Managed'],
```
Comment (ZJvandeWeg) [inline src/_includes/feature_lists/cloud.njk:7]:
```suggestion
            values: ['Self-Managed', 'FlowForge or Self-Managed', 'Flowforge or Self-Managed'],
```

File: src/pricing.njk (hunk 3 above)
Comment (ZJvandeWeg) [reply src/pricing.njk:13]: But than it's makers or practitioners?

File: src/pricing.njk (hunk 4 above)
Comment (ZJvandeWeg) [reply src/pricing.njk:27]: Probably Netlify not deploying the docs. Weird stuff

**Pull request merged**